.gitignore

```diff
@@ -1,8 +1,10 @@
-/target/
-/packaged/
+target/
```

Comment:
This style of entry (with no leading `/`) matches both items at the repository root, and also items within subdirectories - making the wildcard variants below redundant.